### PR TITLE
added gitMinimum.md

### LINKED ISSUE #3095: [Term Entry] Java Calendar getMinimum()
### Reason/inspiration (optional)

Please add the `getMinimum()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar)�class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/getMinimum/getMinimum.md`

The entry should have the following:

�  An introductory paragraph about what the method is and does.
�  A syntax section with a pseudo code block with an explanation of it.
�  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
### Description

Added term entry for java calendar's getMinimum method. Closes #3095 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getMinimum/getMinimum.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.getMinimum()` method of the `Calendar` class returns the minimum value for the given calendar field of this `Calendar` class's instance.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getMinimum/getMinimum.md:15]:
```suggestion
The **`.getMinimum()`** method of the `Calendar` class returns the minimum value for the given calendar field.
```

File: content/java/concepts/calendar/terms/getMinimum/getMinimum.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+Title: '.getMinimum()'
+Description: 'Returns the minimum value for the given calendar field of this Calendar instance.'
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getMinimum/getMinimum.md:3]:
```suggestion
Description: 'Returns the minimum value for the given calendar field.'
```

File: content/java/concepts/calendar/terms/getMinimum/getMinimum.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+The `.getMinimum()` method of the `Calendar` class returns the minimum value for the given calendar field of this `Calendar` class's instance.
+
+## Syntax
+
+```pseudo
+calendar.getMinimum(field)
+```
+
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getMinimum/getMinimum.md:22]: there should be a short sentence describing `field`

File: content/java/concepts/calendar/terms/getMinimum/getMinimum.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+## Example
+
+In the following example, `.getMinimum()` returns the minimum value for the `Calendar.DAY_OF_WEEK` field.
+
+```java
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getMinimum/getMinimum.md:27]: update the indentation in the code to use 2 spaces, not tabs

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ChiragAgg5k I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@ChiragAgg5k LGTM. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@ChiragAgg5k looks good.

**Pull request merged**